st2actions/st2actions/runners/python_action_wrapper.py

```diff
@@ -62,6 +61,8 @@ def __init__(self, pack, file_path, parameters=None, parent_args=None):
             config.parse_args(args=self._parent_args)
         except Exception:
             pass
+        else:
+            db_setup()
```

Comment:
@lakshmi-kannan and I discussed this offline as well. 

I did not much like this approach either and would prefer to better handle the config. I have left a task for myself to write tests for this class where I can take a closer look and fix this logic as well. I feel failures to parse config should now be fail-fast.